app/controllers/facility_statements_controller.rb

```diff
@@ -38,9 +38,9 @@ def create
       flash[:error] = text("no_selection")
     elsif @statement_creator.create
       @statement_creator.send_statement_emails
-      flash[:notice] = text(success_message, accounts: @statement_creator.formatted_account_list).html_safe
+      flash[:notice] = text(success_message, accounts: @statement_creator.formatted_account_list.html_safe).html_safe
```

Comment:
Empirically, no. Somewhat confusingly, the inner `.html_safe` call is actually just marking the account list as BEING html safe (un-escaping it, basically).

From the [text_helper docs](https://github.com/ahorner/text-helpers/blob/master/README.md):

```text and html will escape all arguments passed to it in order to prevent XSS attacks. If you want to pass html content, you should ensure you mark it as .html_safe

Example: text('welcome_user', username) will escape html characters in username

Welcome &lt;b&gt;Bob&lt;/b&gt;
Example: text('welcome_user', username.html_safe) will output html characters in username

Welcome <b>Bob</b>```